server/app/controllers/admin/AdminProgramController.java

```diff
@@ -78,9 +78,26 @@ public Result index(Request request) {
     Optional<CiviFormProfile> profileMaybe = profileUtils.currentUserProfile(request);
     return ok(
         listView.render(
-            programService.getActiveAndDraftProgramsWithoutQuestionLoad(),
+            programService.getInUseActiveAndDraftProgramsWithoutQuestionLoad(),
             questionService.getReadOnlyQuestionServiceSync(),
             request,
+            "In use",
```

Comment:
suggestion (blocking): Make "In use" and "Disabled" into an enum that you can reference in both places. That way you don't have to worry about typing in the string incorrectly in one of the places, and it will be easier to add new program statuses in the future if we need to.

Reply:
Gotcha, added a program tab enum class. Lmk what you thin.